tests/ui/structs/default-field-values/non-exhaustive-ctor-2.stderr

```diff
@@ -0,0 +1,97 @@
+error[E0603]: struct `Priv1` is private
+  --> $DIR/non-exhaustive-ctor-2.rs:19:39
+   |
+LL |     let _ = S { field: (), field1: m::Priv1 {} };
+   |                            ------     ^^^^^ private struct
+   |                            |
+   |                            while setting this field
+   |
+note: the struct `Priv1` is defined here
+  --> $DIR/non-exhaustive-ctor-2.rs:14:4
+   |
+LL |    struct Priv1 {}
+   |    ^^^^^^^^^^^^
+help: the field `field1` you're trying to set has a default value, you can use `..` to use it
```

Comment:
Nit: maybe

```suggestion
help: the type `m::Priv1` of field `field1` is private, but you can construct a default value for the type via `..` in the struct initializer expression
```

or some wording to that effect, since we're constructing a default value, not *using* an already constructed value, right?

Reply:
What do you think of the new wording?

Reply:
That looks good, thanks